components/nodemanager-service/pgdb/filtering.go

```diff
@@ -66,7 +67,7 @@ func buildWhereHavingFilter(mergeableFilters []*common.Filter, tableAbbrev strin
 		return "", "", errors.Wrap(err, "buildWhereHavingFilter error")
 	}
 
-	var conditions []string
+	var conditions []string // nolint: prealloc
```

Comment:
I'm not sure why, but the linter mis-identifies this one.  Someone more familiar with the code might be able to say whether it is likely we see more tagFilters or conditions, but I think just saying 'nolint' is fine for now.